packages/block-editor/src/components/block-actions/index.js

```diff
@@ -75,11 +88,9 @@ export default function BlockActions( {
 	return children( {
 		canCopyStyles,
 		canDuplicate,
-		canInsertDefaultBlock,
+		canInsertBlock,
```

Comment:
Just confirming that `BlockActions` isn't a public API, yeah?

Reply:
Correct. It's not a public API.